lib/screens/home/widgets/category_main_menu.dart

```diff
@@ -15,33 +20,112 @@ class _CategoryMainMenuState extends ConsumerState<CategoryMainMenu> {
     final gridTile = ref
         .read(homeViewModelProvider.notifier)
         .cocktailMenuTiles
-        .map((tile) => _buildGridTile(withName: tile.name))
+        .map((tile) => _buildGridTile(cocktailMenu: tile))
         .toList();
 
-    return Center(
-        child: GridView.count(
-      primary: false,
-      padding: const EdgeInsets.all(20),
-      crossAxisSpacing: 10,
-      mainAxisSpacing: 10,
-      crossAxisCount: 2,
-      children: gridTile,
-    ));
+    return Stack(
+      children: [_mainTitle(), _mainMenu(gridTile)],
+    );
   }
 
-  Widget _buildGridTile({required String withName}) {
-    return GridTile(
-      child: Container(
-        padding: const EdgeInsets.all(8),
-        decoration: BoxDecoration(
-            border: Border.all(color: Colors.amberAccent, width: 5),
-            color: Colors.amber.shade100,
-            borderRadius: const BorderRadius.all(Radius.circular(20))),
-        child: Center(
-            child: Text(
-          withName,
+  Container _mainTitle() {
+    return Container(
+      height: 440,
+      width: 428,
+      decoration: const BoxDecoration(
+          image: DecorationImage(
+              image: AssetImage(Assets.cocktailsBackgroundApp),
+              fit: BoxFit.fill,
+              opacity: 0.6),
+          borderRadius: BorderRadius.only(
+              bottomLeft: Radius.circular(90),
+              bottomRight: Radius.circular(90)),
+          color: CocktailColors.primary),
+      child: Center(
+          child: SizedBox(
+        width: 280,
+        child: Text(
+          ref.read(homeViewModelProvider.notifier).title.toUpperCase(),
+          maxLines: 2,
           textAlign: TextAlign.center,
-        )),
+          style: CocktailsFonts.headTitle,
+        ),
+      )),
+    );
+  }
+
+  Padding _mainMenu(List<Widget> gridTile) {
+    return Padding(
+      padding: const EdgeInsets.only(top: 356),
+      child: Center(
+          child: GridView.count(
+        primary: false,
+        padding: const EdgeInsets.all(20),
+        crossAxisSpacing: 10,
+        mainAxisSpacing: 10,
```

Comment:
Let's add all this double values in the `Spacing` constant file

Reply:
Done, it will be added in a new branch.